examples/dbscan/ArborX_DBSCAN.hpp

```diff
@@ -120,29 +120,45 @@ void dbscan(ExecutionSpace exec_space, Primitives const &primitives,
 
   clock::time_point start_total;
   clock::time_point start;
-  clock::time_point start_local;
-  std::chrono::duration<double> elapsed_construction;
-  std::chrono::duration<double> elapsed_stat;
-  std::chrono::duration<double> elapsed_neigh;
-  std::chrono::duration<double> elapsed_query;
-  std::chrono::duration<double> elapsed_cluster;
+  std::chrono::duration<double> elapsed_construction = clock::duration::zero();
+  std::chrono::duration<double> elapsed_stat = clock::duration::zero();
+  std::chrono::duration<double> elapsed_neigh = clock::duration::zero();
+  std::chrono::duration<double> elapsed_query = clock::duration::zero();
+  std::chrono::duration<double> elapsed_cluster = clock::duration::zero();
   std::chrono::duration<double> elapsed_total = clock::duration::zero();
-  std::chrono::duration<double> elapsed_verify;
+  std::chrono::duration<double> elapsed_verify = clock::duration::zero();
 
-  start_total = clock::now();
+  auto clock_start = [&exec_space, verbose](clock::time_point &clock_time) {
+    if (verbose)
```

Comment:
Why do you have the condition?

Reply:
So that if one is not interested in timers, we don't fence.